randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1749,6 +1765,10 @@ Extra - asset_id: collision_camera_020
   * Morph Ball Tunnel to Energy Recharge Station (Gate)/Tunnel to Energy Recharge Station Secret
   > Tile Group (BOMB WEIGHT)
       Lay Bomb
+  > Corner
```

Comment:
Remove the node for Tile Group (BOMB WEIGHT) too

Reply:
I somehow did not save my last edit...